contrib/texmf/template.py

```diff
@@ -0,0 +1,63 @@
+# keep in sync with texlive
+pkgname = "texmf"
+pkgver = "20240312"
+pkgrel = 0
+pkgdesc = ""  # TODO
+maintainer = "Subhaditya Nath <sn03.general@gmail.com>"
+license = "GPL-3.0-or-later"
+url = "https://tug.org/texlive"
+source = f"ftp://tug.org/historic/systems/texlive/{pkgver[:4]}/texlive-{pkgver}-texmf.tar.xz"
+sha256 = "c8eae2deaaf51e86d93baa6bbcc4e94c12aa06a0d92893df474cc7d2a012c7a7"
+
+
+def post_extract(self):
+    self.rm("texmf-dist/doc/luatex/opbible/txs-gen/mod2tex")
```

Comment:
this needs a comment why it's removed

Reply:
I added a comment.

Could you check if that's satisfactory?

Reply:
.pyc isn't really a binary so what does deleting it fix

Reply:
if we don't delete it, it gets included in the -doc package, and then cbuild complains about .pyc outside __pycache__

Reply:
`# delete binaries from usr/share`